frontend/dataset_details.html

```diff
@@ -540,30 +504,23 @@ <h1><!--Section B ---> Questions</h1>
                                                     Add Link
                                                 </button>
                                             </span>
-                                </div><!-- /input-group -->
+                                </div>
                                 <br />
                             </div>
                             <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
```

Comment:
According to https://getbootstrap.com/docs/3.3/css/#grid-options, `col-xs-12 col-sm-12` should be enough, because `sm` is for `≥768px`, including `md` and `lg`.